composer.json

```diff
@@ -3,6 +3,7 @@
     "license": "MIT",
     "require": {
         "php": ">=5.6.0",
+        "ext-pcntl":"*",
```

Comment:
this is already checked in mkraemer's library?

Reply:
Yes, there are already same check in another library. But why should we relate on it when we have some direct function calls in ProcessManager?

Reply:
Because we already relate on the library?

Reply:
If there are direct function calls, then it should be defined in `composer.json`.